symengine/eval.cpp

```diff
@@ -0,0 +1,52 @@
+#include <exception>
+#include <symengine/eval.h>
+#include <symengine/eval_double.h>
+#include <symengine/real_double.h>
+#include <symengine/complex_double.h>
+#include <symengine/symengine_rcp.h>
+#ifdef HAVE_SYMENGINE_MPFR
+#include <mpfr.h>
+#include <symengine/real_mpfr.h>
+#include <symengine/eval_mpfr.h>
+#endif // HAVE_SYMENGINE_MPFR
+
+#ifdef HAVE_SYMENGINE_MPC
+#include <mpc.h>
```

Comment:
`<mpc.h>` is already there in `<symengine/complex_mpc.h>`. I am not sure if occurrences like these are removed.